extension/scripts/features/jail-filter/ttJailFilter.js

```diff
@@ -108,6 +108,13 @@
 		filterContent.appendChild(scoreFilter.element);
 		localFilters["Score Filter"] = { getStartEnd: scoreFilter.getStartEnd, updateCounter: scoreFilter.updateCounter };
 
+		const bailCostFilter = createTextbox({ type: "number", description: "Bail Cost less than", min: 0 });
+		bailCostFilter.setValue(filters.jail.bailCost || 9999999);

```

Comment:
Seems like a bad default to be honest, as bail cost might be higher than this. Safest would probably just keep it empty as default and have that disable the filter.